data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', 'Feb 3, 2020', 0, true, 10.23 );
+INSERT INTO animals VALUES (2, 'Gabumon', 'Nov 15th, 2018', 2, true, 8.0 );
+INSERT INTO animals VALUES (3, 'Pikachu', 'Jan 7th, 2021', 1, false, 15.04 );
+INSERT INTO animals VALUES (4, 'Devimon', 'May 12th, 2017', 5, true, 11.0 );
```

Comment:
* I get errors when i run this set of queries below is the screenshot of the errors i get HINT 💭 kindly remove the `th` from the days everything would work fine
![image](https://user-images.githubusercontent.com/52497246/183798364-d27a9d0a-724b-470c-8cf0-5b538b5eb9e2.png)